tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -628,6 +631,16 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
         if (!containersStamped || !activeDashboards || !selectedDashboard) {
           return;
         }
+        const previous = this._previousDashboard;
+        this._previousDashboard = selectedDashboard;
+        if (previous && previous != selectedDashboard) {
+          if (tf_tensorboard.dashboardRegistry[previous].shouldRemoveDom) {
+            const div = this.$$(`.dashboard-container[data-dashboard=${previous}]`);
```

Comment:
I'm not sure I fully understand. Please note doing this would likely involve refactoring other parts of this file, like the code beneath this if statement.

Reply:
Ah, I had copy-pasted the original code so it used `${selectedDashboard}` instead of `${previous}` which was my intention.  In the intended form it wouldn't involve any other refactoring, since `div` isn't used later in the function.

Here's the current code:
```js
const div = this.$$(`.dashboard-container[data-dashboard=${previous}]`);
if (div.firstChild) {
  div.firstChild.remove();
}
```

Here's what I meant to suggest as an alternative, which adds `#dashboard` to the selector string in the same way that _selectedDashboardComponent() already does.  The benefit is that we're now targeting the dashboard child explicitly and not relying on it being the firstChild of the div.

```js
const dashboard = this.$$(`.dashboard-container[data-dashboard=${previous}] #dashboard`);
if (dashboard) {
  dashboard.remove();
}
```